src/components/NewAssignment.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { Link } from 'react-router-dom'
+import Cookies from 'js-cookie';
+import Button from '@mui/material/Button';
+import {SERVER_URL} from '../constants.js'
+import { TextField } from '@mui/material';
+
+class NewAssignment extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = {assignName: '', dueDate: '', courseId: 0};
+    };
+
+    handleChange = (event) => {
+        if (event.target.name == "name"){
```

Comment:
== works but will have warning every time, can replace with === here and in the other else ifs

Reply:
fixed triple equal signs